pixi.toml

```
@@ -0,0 +1,173 @@
+[project]
+name = "nbsite"
+channels = ["pyviz/label/dev", "conda-forge"]
+platforms = ["linux-64", "osx-arm64", "osx-64", "win-64"]
+
+[activation.env]
+PYTHONIOENCODING = "utf-8"
+
+[environments]
+default = ["py312", "required", "test", "example", "test-example", "lint", "dev"]
+
+[environments.test-39]
+features = ["py39", "required", "test", "example", "test-example"]
+no-default-feature = true
+
+[environments.test-310]
+features = ["py310", "required", "test", "example", "test-example"]
+no-default-feature = true
+
+[environments.test-311]
+features = ["py311", "required", "test", "example", "test-example"]
+no-default-feature = true
+
+[environments.test-312]
+features = ["py312", "required", "test", "example", "test-example"]
+no-default-feature = true
+
+[environments.test-313]
+features = ["py313", "required", "test", "example", "test-example"]
+no-default-feature = true
+
+[environments.docs]
+features = ["py311", "required", "doc", "example"]
+no-default-feature = true
+
+[environments.build]
+features = ["py311", "required", "build"]
+no-default-feature = true
+
+[environments.lint]
+features = ["lint"]
+no-default-feature = true
+
+[feature.required.dependencies]
+beautifulsoup4 = "*"
+ipykernel = "*"
+jinja2 = "*"
+jupyter_client = "*"
+myst-nb = ">=1.1"
+myst-parser = '>=3'
+nbconvert = "*"
+nbformat = "*"
+nomkl = "*"
+notebook = "*"
+packaging = "*"
+param = '>=1.7.0'
+pillow = "*"
+pip = "*"
+portalocker = "*"
+pydata-sphinx-theme = '>=0.15,<0.17'
+pyviz_comms = "*"
+sphinx = '>=7'
+sphinx-copybutton = "*"
+sphinx-design = "*"
+sphinxext-rediraffe = "*"
+
+[feature.required.tasks]
+install = 'python -m pip install --no-deps --disable-pip-version-check -e .'
+setup-dev = { depends-on = ["install", "lint-install"] }
+sync-git-tags = 'python scripts/sync_git_tags.py nbsite'
+
+[feature.py39.dependencies]
+python = "3.9.*"
+
+[feature.py310.dependencies]
+python = "3.10.*"
+
+[feature.py311.dependencies]
+python = "3.11.*"
+
+[feature.py312.dependencies]
+python = "3.12.*"
+
+[feature.py312.activation.env]
+COVERAGE_CORE = "sysmon"
+
+[feature.py313.dependencies]
+python = "3.13.*"
+
+[feature.py313.activation.env]
+COVERAGE_CORE = "sysmon"
+
+# =================== DEV ===================
+
+[feature.dev.dependencies]
+jupyterlab = "*"
+jupyterlab-myst = "*"
+setuptools_scm = ">=6"
+
+[feature.dev.tasks]
+lab = 'jupyter lab'
+
+# =================== SHARED DEPS ===================
+
+# Dependencies required to run the notebooks
+[feature.example.dependencies]
+bokeh = "*"
+holoviews = "*"
+hvplot = "*"
+matplotlib = "*"
```

```suggestion
matplotlib-base = "*"
```
This pulls in reduced dependencies which I think is OK.